### PR TITLE
note (twice!) that Nextcloud is NOT a mail server

### PR DESCRIPTION
**Pull request by jospoortvliet**
screenshot_20180316_172153
screenshot_20180316_172119

Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
I would also put this (again) in the bullet points, as (I guess) many people only read bullet points to get an impression and do not read the text above it.
(Maybe there the term "client" as suggested is a good idea to not repeat oneself)

**Comment by jospoortvliet** [comment]
Where exactly would you put it as third place? What bullet list do you mean?

**Comment by tobiasKaminsky** [comment]
Hehe..sorry I overread the "Note" part beneath the three points.
So my only idea is to use "Nextcloud Mail Client" instead of "Nextcloud Mail".

**Comment by jospoortvliet** [comment]
Well, I considered that already, but we have:
Nextcloud Mail
Nextcloud Calendar
Nextcloud Contacts
(and later Nextcloud Notes, Nextcloud Deck etc)
Adding 'client' to one of them is pretty ugly... So I'd rather avoid that.

**Comment by tobiasKaminsky** [comment]
Yeah, came also into my mind. Then let us hope that this is enough ;-)

File: page-groupware.php [hunk 1]
```diff
@@ -94,15 +96,15 @@
...
+			<li>Share your address books with your team</li>
 			<li>Sync your contacts with phones and other devices</li>
 			</ul>
 			');?></p>
         </div>
 	</div>
 	<div class="row">
         <div class="col-sm-8 col-sm-offset-2">
-            <p class="section--paragraph revealOnScroll text-center"><?php echo $l->t('Nextcloud Groupware relies on native IMAP and SMTP capabilities. It fluently integrates into your existing infrastructure while also providing its own.');?></p>
+            <p class="section--paragraph revealOnScroll text-center"><?php echo $l->t('Nextcloud fluently integrates into your existing infrastructure while also providing its own. Note that Nextcloud Mail is a client for POP or IMAP servers; Nextcloud does not include a mail server. ');?></p>
```
Comment (MariusBluem) [inline page-groupware.php]: POP and/or IMAP-infrastructure but does not include its own mail server.

Comment (jospoortvliet) [reply page-groupware.php]: Done ;-)

**Pull request merged**